### DISCUSSION TITLE
为什么禁用redis keys命令的呢？

### DISCUSSION THREAD

**Discussion by littlefisher666** [hhyo/Archery#1662] [Q&A] [answered]
https://github.com/hhyo/Archery/commit/0247e01daecc296b80fd4f524c4d93b10bface2d

在这个commit中看到把keys命令给下掉了，这个是什么原因呢？

**Comment by unknowissue** [comment, marked as answer]
keys 是有损性能的高危命令。
keys 使用的方法，可以参考 https://www.runoob.com/redis/keys-keys.html
由此，可以想到性能问题，比如 https://juejin.cn/post/6855129005172097031
https://blog.csdn.net/qq_37102984/article/details/115386745
https://blog.csdn.net/sinat_33718563/article/details/123411377

**Comment by littlefisher666** [comment]
@unknowissue 感谢回复，之前没关注过scan命令，一直使用的keys命令的。去看了下源码是支持scan命令的
